src/internal/testutil/pach_client.go

```diff
@@ -32,3 +32,18 @@ func GetPachClient(t testing.TB) *client.APIClient {
 	}
 	return pachClient
 }
+
+// GetPachClient gets a pachyderm client for use in tests. It works for tests
+// running both inside and outside the cluster. Unlike GetPachClient,
+// multiple calls will return new pach client instances
+func GetNewPachClient(t testing.TB) *client.APIClient {
```

Comment:
I was trying to create something similar to this in this P.R. https://github.com/pachyderm/pachyderm/pull/7126

The idea is to contain logic about how to use minikube (which is just a specific implementation of k8s) into a separate package, and make `testutil` only utility functions which do not allocate resources.  What do you think?

Regardless of the direction there: `GetNew` -> `New`

Reply:
So right now we get PachClients from `minikubetestenv`, but we could also get PachClients from an in memory `testpachd` package.  `testutil` continues to be useful even after that switch.